### ISSUE TITLE
Is there some bug in TensorRT?

### ISSUE THREAD

**Issue by bujianyiwang** [NVIDIA-AI-IOT/Lidar_AI_Solution#271]
[stdbuf-1] [NVINFER LOG]: 1: [runtime.cpp::parsePlan::314] Error Code 1: Serialization (Serialization assertion plan->header.magicTag == rt::kPLAN_MAGIC_TAG failed.)
[stdbuf-1] Failed to deserialize engine: /workspace/test/shan.li/install/radar_img_fusion/share/radar_img_fusion/model/20240116/CPC_FCR02.a/resnet50/build/camera.backbone.plan
[stdbuf-1] Failed to create camera backbone.
[stdbuf-1] [2024-07-26 15:04:00.553] [err] [ main ] Failed to create bevfusion core!

TensorRT 8.6.2
CUDA 12.2
Protobuf 3.61

**Comment by hopef** [comment]
No, this is a bug caused by the wrong configuration of LD_LIBRARY_PATH.
